pkgs/coverage/test/collect_coverage_config_test.dart

```diff
@@ -2,36 +2,41 @@ import 'package:coverage/src/coverage_options.dart';
 import 'package:path/path.dart' as path;
 import 'package:test/test.dart';
 
-import 'test_util.dart';
+import '../bin/collect_coverage.dart' as collect_coverage;
+import '../bin/format_coverage.dart' as format_coverage;
+import '../bin/test_with_coverage.dart' as test_with_coverage;
 
 void main() {
+  final formatCoverageArgs = ['--in=./test/collect_coverage_config_test.dart'];
+
   group('defaults when no yaml or command-line args provided', () {
     // Setup
     final defaults = CoverageOptionsProvider().coverageOptions;
 
     test('collect coverage', () {
-      final collectedCoverage = parseArgsCollectCoverage([], defaults);
+      final collectedCoverage = collect_coverage.parseArgs([], defaults);
 
-      expect(collectedCoverage.scopeOutput, defaults.scopeOutput);
+      expect(collectedCoverage.scopedOutput, defaults.scopeOutput);
       expect(collectedCoverage.functionCoverage, defaults.functionCoverage);
       expect(collectedCoverage.branchCoverage, defaults.branchCoverage);
-      expect(collectedCoverage.outputDirectory, 'stdout');
+      expect(collectedCoverage.out, isNull);
     });
 
     test('format coverage', () {
-      final formattedCoverage = parseArgsFormatCoverage([], defaults);
+      final formattedCoverage =
+          format_coverage.parseArgs(formatCoverageArgs, defaults);
 
-      expect(formattedCoverage.outputDirectory, 'stdout');
-      expect(formattedCoverage.packageDirectory, defaults.packageDirectory);
+      expect(formattedCoverage.output, isNull);
+      expect(formattedCoverage.packagePath, defaults.packageDirectory);
     });
 
     test('test with coverage', () async {
-      final testCoverage = await parseArgsTestWithCoverage([], defaults);
+      final testCoverage = await test_with_coverage.parseArgs([], defaults);
 
-      expect(testCoverage.packageDirectory,
-          getAbsolutePath(defaults.packageDirectory));
+      expect(testCoverage.packageDir.toLowerCase(),
```

Comment:
Use `path.canonicalize` here and below. `toLowerCase()` will give the wrong result if someone tries to run these tests on a non-windows machine and has a capital letter in the path to their copy of this repo.